java/client/src/main/java/glide/api/commands/HashBaseCommands.java

```diff
@@ -237,6 +237,23 @@ public interface HashBaseCommands {
      */
     CompletableFuture<String[]> hkeys(String key);
 
+    /**
+     * Returns the string length of the value associated with <code>field</code> in the hash stored at
+     * <code>key</code>.
+     *
+     * @see <a href="https://valkey.io/commands/hstrlen/">redis.io</a> for details.
```

Comment:
Same as above, can do this now or later, either way
```suggestion
     * @see <a href="https://valkey.io/commands/hstrlen/">valkey.io</a> for details.
```